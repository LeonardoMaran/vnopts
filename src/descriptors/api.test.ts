import { apiDescriptor } from './api';

test('apiDescriptor', () => {
  expect(apiDescriptor.value({})).toMatchSnapshot();
  expect(apiDescriptor.value('key')).toMatchSnapshot();
  expect(apiDescriptor.value({ key: 'value' })).toMatchSnapshot();
});
