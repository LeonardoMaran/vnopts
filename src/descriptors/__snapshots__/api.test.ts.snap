// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`apiDescriptor 1`] = `"{}"`;

exports[`apiDescriptor 2`] = `"\\"key\\""`;

exports[`apiDescriptor 3`] = `"{ key: \\"value\\" }"`;
